test/framework/src/main/java/org/elasticsearch/indices/analysis/AnalysisFactoryTestCase.java

```diff
@@ -228,6 +228,8 @@ private static String toCamelCase(String s) {
         // fake term frequency feature (LUCENE-7854)
         .put("delimitedtermfrequency",    Void.class)
 
+        .put("fixedshingle", Void.class)
```

Comment:
please put a comment with something like `not meant to be exposed as-is, will be used internally to index shingles and speed up phrase queries`

Reply:
I added a comment